theano/gpuarray/tests/config.py

```diff
@@ -17,7 +17,7 @@
 
 if theano.config.mode == 'FAST_COMPILE':
     mode_with_gpu = theano.compile.mode.get_mode('FAST_RUN').including('gpuarray').excluding('gpu')
-    mode_without_gpu = theano.compile.mode.get_mode('FAST_RUN').excluding('gpuarray')
+    mode_without_gpu = theano.compile.mode.get_mode('FAST_RUN').excluding('gpuarray', 'gpu')
 else:
     mode_with_gpu = theano.compile.mode.get_default_mode().including('gpuarray').excluding('gpu')
-    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpuarray')
+    mode_without_gpu = theano.compile.mode.get_default_mode().excluding('gpuarray', 'gpu')
```

Comment:
I think those 2 changes aren't good. Why did you need them? This would enable the old back-end at the same time as the new back-end.